loader_hub/gmail/README.md

```diff
@@ -0,0 +1,68 @@
+# Gmail Loader
+
+This loader seaches your Gmail account and parses the resulting emails into `Document`s. The search query can include normal query params, like `from: email@example.com label:inbox`.
+
+As a prerequisite, you will need to register with Google and generate a `credentials.json` file in the directory where you run this loader. See [here](https://developers.google.com/workspace/guides/create-credentials) for instructions.
+
+## Usage
+
+To use this loader, you simply need to pass in a search query string.
+
+```python
+from llama_index import download_loader
+
+GoogleDocsReader = download_loader('GmailReader')
+loader = GoogleDocsReader(query="from: me label:inbox")
+documents = loader.load_data()
+```
+
+## Examples
+
+This loader is designed to be used as a way to load data into [LlamaIndex](https://github.com/jerryjliu/gpt_index/tree/main/gpt_index) and/or subsequently used as a Tool in a [LangChain](https://github.com/hwchase17/langchain) Agent.
+
+### LlamaIndex
+
+```python
+from llama_index import GPTSimpleVectorIndex, download_loader
+
+GoogleDocsReader = download_loader('GmailReader')
```

Comment:
nit: Gmail (also if you want you could just remove the LlamaIndex and Langchain sections to make the readme more concise)